.github/workflows/check.yml

```diff
@@ -0,0 +1,50 @@
+name: check
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ "*" ]
+
+jobs:
+  print_hip_info:
```

Comment:
Neat didn't realize it would build targets without test dependencies.

No computer for a bit so feel free to update it if you're in a hurry - otherwise I'll get to it sometime this weekend.

Reply:
done